dev/dags/example_cosmos_cleanup_dir_dag.py

```diff
@@ -0,0 +1,38 @@
+"""
+We've observed users who had dbt project directories containing symbolic links to files that no longer existed.
+
+Although this issue was not created by Cosmos itself, since this issue was already observed by two users, we thought it
+was useful to give an example DAG illustrating how to clean the problematic directories.
+
+Assuming the cause of the issue no longer exists, this DAG can be run only once.
+"""
+
+# [START dirty_dir_example]
```

Comment:
It would be great if we could use tag `dirty_dir_example` to render this example in docs